improver/wxcode/wxcode_decision_tree.py

```diff
@@ -343,56 +359,56 @@ def wxcode_decision_tree():
             'fail': 'light_precipitation',
-            'probability_thresholds': [0.5, 0.5],
+            'probability_thresholds': [0.5],
             'threshold_condition': '>=',
-            'condition_combination': 'OR',
+            'condition_combination': '',
             'diagnostic_fields': [
-                'probability_of_rainfall_rate_in_vicinity_above_threshold',
-                'probability_of_lwe_snowfall_rate_in_vicinity_'
-                'above_threshold'],
-            'diagnostic_thresholds': [(0.1, 'mm hr-1'),
-                                      (0.1, 'mm hr-1')],
-            'diagnostic_conditions': ['above', 'above']},
+                'probability_of_lwe_precipitation_rate_in_vicinity_'
+                'above_threshold',
+                ],
+            'diagnostic_thresholds': [(0.1, 'mm hr-1')],
+            'diagnostic_conditions': ['above']},
 
+        # B.a
         'sleet_in_vicinity': {
             'succeed': 17,
             'fail': 'rain_or_snow_in_vicinity',
             'probability_thresholds': [0., 0.],
-            'threshold_condition': '>=',
+            'threshold_condition': '>',
             'condition_combination': 'AND',
-            'diagnostic_fields': [
-                ['probability_of_lwe_snowfall_rate_in_vicinity_'
-                 'above_threshold',
-                 'probability_of_rainfall_rate_in_vicinity_above_threshold'],
-                ['probability_of_rainfall_rate_in_vicinity_above_threshold',
-                 'probability_of_lwe_snowfall_rate_in_vicinity_'
-                 'above_threshold']],
-            'diagnostic_gamma': [0.7, 1.0],
+            'diagnostic_fields':
+                [['probability_of_rainfall_rate_above_threshold',
+                  'probability_of_lwe_sleetfall_rate_above_threshold'],
+                 ['probability_of_lwe_snowfall_rate_above_threshold',
+                  'probability_of_lwe_sleetfall_rate_above_threshold']],
+            'diagnostic_gamma': [1.0, 1.0],
             'diagnostic_thresholds': [[(0.1, 'mm hr-1'),
                                        (0.1, 'mm hr-1')],
                                       [(0.1, 'mm hr-1'),
                                        (0.1, 'mm hr-1')]],
             'diagnostic_conditions': [['above', 'above'],
                                       ['above', 'above']]},
 
+        # B.a.a
         'rain_or_snow_in_vicinity': {
             'succeed': 'snow_in_vicinity_cloud',
             'fail': 'rain_in_vicinity_cloud',
             'probability_thresholds': [0.],
             'threshold_condition': '>=',
             'condition_combination': '',
             'diagnostic_fields': [
-                ['probability_of_lwe_snowfall_rate_in_vicinity_'
-                 'above_threshold',
-                 'probability_of_rainfall_rate_in_vicinity_above_threshold']],
+                ['probability_of_lwe_snowfall_rate_above_threshold',
+                 'probability_of_rainfall_rate_above_threshold']],
```

Comment:
What's the distinction between:
```
['probability_of_lwe_snowfall_rate_above_threshold',
 'probability_of_rainfall_rate_above_threshold']
```
and
```
[['probability_of_lwe_snowfall_rate_above_threshold',
  'probability_of_rainfall_rate_above_threshold']]
```
in this context?

Reply:
Also vicinity-wise: is this still a valid query, since these vicinity diagnostics no longer exist and you can only diagnose this from precipitation in vicinity?  (It's possible this will be clearer once I've seen the documentation.)

Reply:
Firstly, there's no difference between single and double square brackets. This is what another ticket has been created to straighten out. I have set single brackets where I have made changes, and left the others alone.
Secondly, I think this query is still valid. We're using the precip-in-vicinity data to determine whether a shower symbol is appropriate and the phase(-at-point) data to determine the phase of the symbol. My assumption is that while the probability of precipitation at a point is too low to trigger the precipitation symbol logic, it is still non-zero if the precipitation-in-vicinity data is high enough to trigger its precipitation symbol logic. Therefore it can still be used to diagnose the phase of the symbol.
I'm not happy with how the precip-in-vicinity logic is being triggered ahead of the light-precip-at-point logic, but I deemed it out-of-scope in this PR.

Reply:
Correction to the first point. I have looked at the other ticket I mentioned more closely and the logic that is going on here. Where double-brackets are in place, the logic is going to make a decision based on which of the two probability fields is larger at each point. It does this by subtracting one from the other (with a gamma value of 1 indicating that both fields are equally weighted) and then comparing with a threshold of zero.
Where single brackets are in use, two separate conditions are created and combined using AND or OR logic.

Reply:
That works.  Thanks for the explanation.